a/arrow-Go/build_info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "arrow",
```

Comment:
@vinodk99 Dont we need package name also like "arrow-go"?
cc : @seth-priya

Reply:
@nikhil-kalbande ,since it is a arrow package we are building & testing for go module in that arrow repository.That's why we need to add the arrow as a package name.

Reply:
you have mentioned arrow-go as package name in build script top. So i had this doubt.
Package name and package folder name should be same